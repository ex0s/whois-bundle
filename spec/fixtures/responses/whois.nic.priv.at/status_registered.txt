
% Rights restricted by copyright and data protection act.
% See http://www.nic.priv.at/ for details.

domain:      nic.priv.at
descr:       Network Information Center for priv.at
admin-c:     HM-PRIVAT
tech-c:      HM-PRIVAT
zone-c:      HM-PRIVAT
notify:      hostmaster@nic.priv.at
mnt-by:      PRIVAT-MNT
changed:     cm@nic.priv.at 20021013
source:      PRIVAT

role:        Hostmaster priv.at
address:     Verein fuer Internet-Benutzer Oesterreichs (VIBE!AT)
address:     c/o Gerald Pfeifer
address:     Mondweg 64
address:     A-1140
address:     Wien
address:     Austria
e-mail:      hostmaster@nic.priv.at
admin-c:     CM125
tech-c:      GP362-PRIVAT
tech-c:      TM318-PRIVAT
tech-c:      RW1
tech-c:      CM125
tech-c:      OL122
nic-hdl:     HM-PRIVAT
notify:      hostmaster@nic.priv.at
mnt-by:      PRIVAT-MNT
changed:     cm@nic.priv.at 20021013
source:      PRIVAT


