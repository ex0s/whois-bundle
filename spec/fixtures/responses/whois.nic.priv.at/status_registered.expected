#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %ERROR{AttributeNotSupported}

#updated_on
  %s %CLASS{time}
  %s %TIME{2002-10-13}

#expires_on
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %ERROR{AttributeNotSupported}
